gulpfile.js

```diff
@@ -122,37 +122,24 @@ gulp.task('build:backend:sass:generate-css', function () {
     .pipe(livereload())
 })
 
-gulp.task('build:backend', function () {
-  gulp.start(
-    'build:backend:assets:copy-css-vendors',
-    'build:backend:assets:copy-fonts-vendors',
-    'build:backend:assets:copy-js-vendors',
-    'build:backend:assets:copy-fine-uploader-css-and-images',
-    'build:backend:sass:generate-css',
-    'build:backend:assets:copy-ckeditor'
-  )
-})
+const buildBackend = gulp.parallel(
+  'build:backend:assets:copy-css-vendors',
+  'build:backend:assets:copy-fonts-vendors',
+  'build:backend:assets:copy-js-vendors',
+  'build:backend:assets:copy-fine-uploader-css-and-images',
+  'build:backend:sass:generate-css',
```

Comment:
Zeker dat die in parallel mogen gedaan worden. 
As in: heeft `build:backend:sass:generate-css` bijvoorbeeld geen zaken nodig die in `build:backend:assets:copy-css-vendors` gebeuren?

Reply:
Ja, die mogen parallel gedaan worden. De files die overgezet worden in `build:backend:assets:copy-css-vendors` zijn `.css` files en hebben dus niet te maken met compilen van sass.

Reply:
👍